src/event-stream/event-server.ts

```diff
@@ -722,6 +723,25 @@ export async function startEventServer(opts: {
   );
 
   app.use(bodyParser.json({ type: 'application/json', limit: '500MB' }));
+
+  if (process.env.IBD_MODE_UNTIL_BLOCK) {
+    app.use(['/new_mempool_tx', '/drop_mempool_tx', '/new_burn_block'], async (req, res, next) => {
+      try {
+        const chainTip = await db.getChainTip(db.sql, false);
+        if (
+          process.env.IBD_MODE_UNTIL_BLOCK &&
+          chainTip.blockHeight > Number.parseInt(process.env.IBD_MODE_UNTIL_BLOCK)
+        ) {
+          next();
+        } else {
+          res.send(`${req.originalUrl} is not available while IBD mode is active.`);
```

Comment:
You should only send a blank `200` status response in this case

Reply:
done